package.json

```diff
@@ -16,12 +16,11 @@
     "typescript": "^4.3.5"
   },
   "devDependencies": {
-    "@mate-academy/eslint-config-react-typescript": "*",
     "@mate-academy/eslint-config-react": "*",
-    "@mate-academy/scripts": "^0.9.0",
+    "@mate-academy/eslint-config-react-typescript": "*",
+    "@mate-academy/scripts": "^0.9.13",
```

Comment:
Don't add this

Reply:
don't add package.json and package-lock to PR